src/processes.c

```diff
@@ -2029,6 +2029,8 @@ static int ps_read (void)
 	int stopped  = 0;
 	int paging   = 0;
 	int blocked  = 0;
+	int num_cmdline  = 0;
+	int count = 0;                /* returns number of processes */
```

Comment:
Nit: this doesn't "return" anything. How about `total_processes`? And s/`num_cmdline`/`cmdline_processes`/?
I see this in other parts of this code too -- would suggest renaming those as well.

Reply:
Done.